.azure/templates/build-config-user.yml

```diff
@@ -28,13 +28,13 @@ jobs:
     inputs:
       pwsh: true
       filePath: scripts/build.ps1
-      arguments: -Arch ${{ parameters.arch }} -Tls ${{ parameters.tls }} ${{ parameters.extraBuildArgs }}
+      arguments: -Arch ${{ parameters.arch }} -Tls ${{ parameters.tls }} -Platform ${{ parameters.platform }} ${{ parameters.extraBuildArgs }}
```

Comment:
Getting closer. Again, sorry to keep asking for more. I don't care so much for a `-Platform` parameter that's only used for UWP. I liked the `-UWP` way at the PowerShell script layer. Somehow I'd like to turn a `parameters.platform == UWP` into that other param... Ideas?

Reply:
Only way to do that is to create another script azure script specifically for uwp, or to conventionalize both build source code tasks in build-config-user.yml.

The platform can be nice though, because we could use that same flag for cross compilation in the future, specifically for like Pi builds.

Reply:
Ok.